app/models/ticket_log_change.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class TicketLogChange < ApplicationRecord
+  enum :field_name, {
+    status: "status",
+  }
```

Comment:
There will definitely be fields (for specific metadata of specific ticket flows like nationality change or competition announcement) that cannot be put into a general-purpose enum. It's the same overall thinking as my comments on the `action_type` above.

Reply:
Same comment here - the `field_name` will be `metadata`, and we can have another column `metadata_field` to say which metadata column of the ticket was changed.

(`status` is an exception - even though it's in `metadata`, we can consider it as ticket-specific)

Reply:
Agreed, based on my other comments above.